rotkehlchen/chain/ethereum/modules/zksync/decoder.py

```diff
@@ -53,10 +53,10 @@ def _decode_deposit(self, context: DecoderContext, user_address: ChecksumEvmAddr
         for event in context.decoded_events:
             if event.event_type == HistoryEventType.SPEND and event.location_label == user_address:
                 resolved_event_asset = event.asset.resolve_to_crypto_asset()
-                event_raw_amount = asset_raw_value(
+                event_raw_amount = normalized_fval_value_decimals(
                     amount=event.balance.amount,
-                    asset=resolved_event_asset,
-                )
+                    decimals=-get_decimals(resolved_event_asset),
```

Comment:
why is there a `-` here?

Reply:
this does the opposite. instead of `/ 10**18` it does `/10**-18 = * 10**-18`. What the function wants is to check that `1800000 == 1.8 * 10**6`

Reply:
I don't get it :sweat_smile: